Stable-Branch-Strategy.md

```diff
@@ -80,19 +80,23 @@ stable and master. While this is not in place currently, it should be considered
 
 ### Patch releases
 
-Releases are made on a weekly cadence for patch releases, which include a GitHub release as
+Releases are normally made every-other-week for patch releases, which include a GitHub release as
 well as binary packages. These patch releases are made for both stable branches, and a 'release candidate'
 for the next `MAJOR` or `MINOR` is created from master. If there are no changes across all the repositories, no
 release is created and an announcement is made on the developer mailing list to highlight this.
 If a release is being made, each repository is tagged for this release, regardless
-of whether changes are introduced.
+of whether changes are introduced. Release schedule can be seen on the
```

Comment:
s/Release schedule/The release schedule/ (here and below)?

Reply:
agreed. done